dashboard/forms.py

```diff
@@ -371,26 +370,15 @@ def __init__(self, *args, **kwargs):
             self.fields['remove_brothers'].queryset = brothers_list.filter(id__in=eligible_attendees)
 
 
-class BrotherAttendanceForm(forms.Form):
-    present = forms.BooleanField(label="", required=False, label_suffix='')
-
-    def __init__(self, *args, **kwargs):
-        brother = kwargs.pop('brother', "")
-        super(BrotherAttendanceForm, self).__init__(*args, **kwargs)
-
-        if brother:
-            self.fields['present'].label = brother
-
-
-class PnmAttendanceForm(forms.Form):
+class AttendanceForm(forms.Form):
     present = forms.BooleanField(label="", required=False, label_suffix='')
 
     def __init__(self, *args, **kwargs):
-        pnm = kwargs.pop('pnm', "")
-        super(PnmAttendanceForm, self).__init__(*args, **kwargs)
+        person = kwargs.pop('person', "")
```

Comment:
attendee might be a better variable name instead of person. Person seems extremely vague